packages/components/src/Icon/Icon.js

```diff
@@ -10,6 +10,7 @@ function Icon(props, forwardedRef) {
 		children,
 		className,
 		color,
+		fill = color,
```

Comment:
`@wordpress/icons` require `fill` rather than `color`. This may be a hack, not sure, will leave final approval for this solution up to @ItsJonQ

Reply:
Hmm 🤔 . This one is an interesting one. Unlike other components (like `Button`), Gutenberg's `Icon` comes from a non `@wordpress/components` package.

For a second, I thought, "Well... what if we just used `Icon` from `@wordpress/icons` instead".
But then I realized that the G2 `Icon` component has many advantages, such as it being powered by `View` and it being tapped into the Style + Context system.

Ultimately, because we're using `@wordpress/icons` **svg** code, we have to accommodate the wrapper (that being `Icon`) for that code. For example, [react-icons](https://github.com/react-icons/react-icons) SVGs are built to use `color`.

Based on this, I think we should go with `fill`.

If we do, we can remove the `color` prop from this `Icon` component.

@saramarcondes Lemme know what you think ❤️

Reply:
Sounds good to me!